Observer/AdyenCcDataAssignObserver.php

```diff
@@ -40,6 +40,11 @@ class AdyenCcDataAssignObserver extends AbstractDataAssignObserver
     const ENCRYPTED_EXPIRY_YEAR = 'expiryYear';
     const HOLDER_NAME = 'holderName';
     const VARIANT = 'variant';
+    const JAVA_ENABLED = 'java_enabled';
+    const SCREEN_COLOR_DEPTH = 'screen_color_depth';
+    const SCREEN_WIDTH = 'screen_width';
+    const SCREEN_HEIGHT = 'screen_height';
+    const TIMEZONE_OFFSET = 'timezone_offset';
```

Comment:
I am seeing some fields like channel and origin not present. Not sure if you will add it in next story. Just thought to mention so that we don't miss it.

Reply:
Good one. Actually I retrieve the origin from the frontend and not from the frontend. Let's discuss if we should grab everything from the frontend or if it is more secure to get from backend. I have already had the functions to retrieve these details from the backend so it was also easier for me.